mars/deploy/oscar/session.py

```diff
@@ -1174,7 +1181,14 @@ async def fetch_infos(self, *tileables, fields, **kwargs) -> list:
         if kwargs:  # pragma: no cover
             unexpected_keys = ", ".join(list(kwargs.keys()))
             raise TypeError(f"`fetch` got unexpected arguments: {unexpected_keys}")
-
+        # following fields needs to access storage API to get the meta.
+        _need_query_storage_fields = {"level", "memory_size", "store_size"}
+        _need_query_storage = bool(_need_query_storage_fields & fields)
+        get_chunk_meta_fields = ["bands"]
+        if not _need_query_storage:
+            get_chunk_meta_fields = list(fields)
+            if "band" in get_chunk_meta_fields:
+                get_chunk_meta_fields[get_chunk_meta_fields.index("band")] = "bands"
```

Comment:
Why changing the `band` to `bands`? Can we just replace it by a list comprehension?